test/tpu/xla_test_job.yaml

```diff
@@ -25,9 +38,9 @@ spec:
     image: gcr.io/$PROJECT_ID/pytorch-xla-test:$BUILD_ID
     command:
     - bash
-    - -c
+    - -cxe
     - |
-      python3 pytorch/xla/test/pjrt/test_operations.py -v
```

Comment:
before the change, wouldn't it throw a file not found error?

Reply:
Right, but the failing status was ignored. That's fixed by setting the `-e` flag in the bash script

Reply:
thanks!